problem3.py

```diff
@@ -0,0 +1,40 @@
+"""
+time: O(n)
+space:O(n)
+Leet: accepted 79.86 percentile
```

Comment:
what does this mean?

Reply:
Means that my code surpassed roughly 80% of the submitted python codes on Leetcode

Reply:
ohh that doesnt matter
it depends on the testcases 
we dont need to bother about that